### PR TITLE
Set `accepted_at` when an auction is accepted

### PR DESCRIPTION
**Pull request by jessieay**
* Show timestamp on admin/auctions#show
* This data will help us with our stats page
* Part of https://github.com/18F/micropurchase/issues/705

### CODE REVIEW

File: db/migrate/20160627225519_add_accepted_at_to_auctions.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class AddAcceptedAtToAuctions < ActiveRecord::Migration
+  def up
+    add_column :auctions, :accepted_at, :datetime
+
+    execute <<-SQL
+      UPDATE auctions
+      SET accepted_at = delivery_due_at;
```
Comment (harrisj) [inline db/migrate/20160627225519_add_accepted_at_to_auctions.rb]: Does this set accepted_at even for auctions that were not accepted? Maybe add a `where accepted = true`

Comment (jessieay) [inline db/migrate/20160627225519_add_accepted_at_to_auctions.rb]: good call!

Comment (jessieay) [inline db/migrate/20160627225519_add_accepted_at_to_auctions.rb]: updated at dfab52b

**Comment by harrisj** [comment]
Perfect. Let's merge!

**Pull request merged**